lib/modules/showImages.js

```diff
@@ -1403,21 +1415,17 @@ function keepMediaVisible(media) {
 	});
 }
 
-function setMediaClippyText(elem) {
-	if (!module.options.clippy.value || elem.title) return;
-
+function getClippyText() {
 	const clippy = [];
 	if (module.options.imageZoom.value) {
 		clippy.push('drag to resize');
 	}
 
-	if (elem.tagName === 'IMG' && module.options.imageMove.value) {
+	if (module.options.imageMove.value) {
 		clippy.push('shift-drag to move');
 	}
 
-	const title = clippy.join(' or ');
-
-	elem.setAttribute('title', title);
```

Comment:
:+1: to showing clippy inside media controls, or not at all. It's always seemed more obnoxious than useful as a tooltip, at least from the feedback I've heard.